TestServer_ConcurrentPointsWriter_Subscriber has inconsistent results

### Bug report

__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
branch 1.5

__Steps to reproduce:__

```
$ go test -c -race ./tests
$ while ./tests.test -test.run=_ConcurrentPointsWriter_; do true; done
PASS
PASS
...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x90 pc=0x1c5039c]

goroutine 53 [running]:
github.com/influxdata/influxdb/tests.(*LocalServer).WritePoints(0xc4202211d0, 0x1f954d5, 0x3, 0x1f955c8, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /Users/mr/go/src/github.com/influxdata/influxdb/tests/server_helpers.go:347 +0xbc
github.com/influxdata/influxdb/tests.TestServer_ConcurrentPointsWriter_Subscriber.func1(0xc420258a80, 0x20f00e0, 0xc4202211d0)
    /Users/mr/go/src/github.com/influxdata/influxdb/tests/server_test.go:9173 +0x1b1
created by github.com/influxdata/influxdb/tests.TestServer_ConcurrentPointsWriter_Subscriber
    /Users/mr/go/src/github.com/influxdata/influxdb/tests/server_test.go:9163 +0x134
```

__Expected behavior:__  PASS

__Actual behavior:__  Occasional failure

--- a/tests/server_test.go
+++ b/tests/server_test.go
@@ -9158,7 +9158,10 @@ func TestServer_ConcurrentPointsWriter_Subscriber(t *testing.T) {
 	}
 	// goroutine to write points
 	done := make(chan struct{})
+	var wg sync.WaitGroup
+	wg.Add(1)
 	go func() {
+		defer wg.Done()
 		for {
 			select {
 			case <-done:
@@ -9176,6 +9179,7 @@ func TestServer_ConcurrentPointsWriter_Subscriber(t *testing.T) {
 	time.Sleep(10 * time.Millisecond)
 
 	close(done)
+	wg.Wait()
 }
 
 // Ensure time in where clause is inclusive

